src/screens/notifications-settings/notifications-settings.selectors.ts

```diff
@@ -1,3 +1,3 @@
 export enum NotificationsSettingsSelectors {
-  NewsSwitch = 'News Switch on Notifications page is Settings'
+  newsSwitch = 'Settings Notifications/News Toggle'
```

Comment:
different names: "newsSwitch" and "News Toggle"

Reply:
Toggle